instrumentation/opentelemetry-instrumentation-falcon/tests/test_falcon.py

```diff
@@ -124,20 +126,28 @@ def _test_method(self, method):
                 SpanAttributes.HTTP_SCHEME: "http",
                 SpanAttributes.NET_HOST_PORT: 80,
                 SpanAttributes.HTTP_HOST: "falconframework.org",
-                SpanAttributes.HTTP_TARGET: "/",
                 SpanAttributes.NET_PEER_PORT: 65133,
                 SpanAttributes.HTTP_FLAVOR: "1.1",
                 "falcon.resource": "HelloWorldResource",
                 SpanAttributes.HTTP_STATUS_CODE: 201,
             },
         )
         # In falcon<3, NET_PEER_IP is always set by default to 127.0.0.1
-        # In falcon>3, NET_PEER_IP is not set to anything by default to
+        # In falcon>=3, NET_PEER_IP is not set to anything by default
         # https://github.com/falconry/falcon/blob/5233d0abed977d9dab78ebadf305f5abe2eef07c/falcon/testing/helpers.py#L1168-L1172 # noqa
         if SpanAttributes.NET_PEER_IP in span.attributes:
             self.assertEqual(
                 span.attributes[SpanAttributes.NET_PEER_IP], "127.0.0.1"
             )
+        # In falcon<3.1.2, HTTP_TARGET is always set to / in TestClient
```

Comment:
what do you think on adding a property to test class like

```
@property
def _has_fixed_http_target(self):
    # In falcon<3.1.2, HTTP_TARGET is always set to / in TestClient
    # In falcon>=3.1.2, HTTP_TARGET is set to unencoded path by default
    # https://github.com/falconry/falcon/blob/69cdcd6edd2ee33f4ac9f7793e1cc3c4f99da692/falcon/testing/helpers.py#L1153-1156 # noqa
    return _parsed_falcon_version < package_version.parse("3.1.2")
```

So that you can keep the assert with the others and don't repeat the comment?
```
SpanAttributes.HTTP_TARGET: "/" if self._has_fixed_http_target else "/hello".
```

Reply:
Thanks. I want this for py313 PR, so I fixed it